### PR TITLE
Add Artsy

### PR DESCRIPTION
**Pull request by cchitsiang**
Company website: https://www.artsy.net/
Github link: https://github.com/artsy
Description: A company love open source, which maintain a philosophy of keeping their work Open by Default.

### CODE REVIEW

**Comment by Mayeu** [comment]
Hello :wave:

Thank you for opening this pull request :pray: I took a look at the website and the various repos to get a feel of the project. So Artsy is like an online gallery allowing you to explore the arts of museum, and also bid/buy via gallery?

I am really unsure about the museum part, do you know what it does and if this part is also F/LOSS?

To me it looks like it matches the requirement to be in the list, with the caveat that running your own instance does not really enable you to do anything since the value is in the data that Artsy has accumulated, and their partnerships.

I would love to have your view on that @cchitsiang, and the view of other watchers of this repo.

**Comment by cchitsiang** [comment]
I guess this well explained what is Artsy about: https://github.com/artsy/meta/blob/master/meta/what_is_artsy.md

**Comment by agarrharr** [comment]
Does Artsy release all of the software it makes as open source?

**Comment by Mayeu** [comment]
@cchitsiang: Thank you for the link :+1:
@agarrharr: They have the [following page listing all their F/LOSS](https://artsy.github.io/open-source/), and from the list it seems that all parts of the website and the web app are open. They also point [to this blogpost](http://code.dblock.org/2015/02/09/becoming-open-source-by-default.html) from there.

I am fine with the inclusion of Artsy in the list :+1:

File: README.md [hunk 1]
```diff
@@ -35,6 +35,7 @@ Please help me grow this list :) You could:
 - :moneybag: [Sentry](https://sentry.io), [$10.5M](https://www.crunchbase.com/organization/sentry#section-overview), [on GitHub](https://github.com/getsentry): real-time crash reporting for your web apps, mobile apps, and games.
 - :boot: [Spectrum](https://spectrum.chat/), [on GitHub](https://github.com/withspectrum/spectrum): the community platform for the future.
 - :boot: [Zulip](https://zulipchat.com), [on GitHub](https://github.com/zulip/zulip): the world's most productive group chat
+- :boot: [Artsy](https://www.artsy.net), [on Github](https://github.com/artsy): the online platform for art collecting and education, which maintain a philosophy of keeping their work [Open by Default](http://artsy.github.io/open-source/).
```
Comment (Mayeu) [inline README.md]: I just realise that the entry should be sorted alphabetically, can you fix that? And then I'll merge it :+1:

Comment (Mayeu) [inline README.md]: I just realise that the entry should be sorted alphabetically, can you fix that? And then I'll merge it :+1:

**Review by Mayeu — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (cchitsiang) [reply README.md]: Sure

**Review by Mayeu — APPROVED** [review]

**Comment by Mayeu** [comment]
Thank you for your contribution :pray:

**Pull request merged**